CONTRIBUTING.md

```diff
@@ -0,0 +1,619 @@
+This page outlines how you can provide feedback and contribute to TinyMCE documentation.
+
+# Contributor Code of Conduct
+
+For our Contributor Code of Conduct, visit our [Contributor Covenant Code of Conduct on GitHub](https://github.com/tinymce/tinymce-docs/blob/develop/CODE_OF_CONDUCT.md#contributor-covenant-code-of-conduct).
+
+# How to contribute to TinyMCE’s documentation
+
+We welcome people with technical writing experience to help make TinyMCE’s docs as useful to our community as possible. Although you don’t need to be an expert developer to contribute, a basic understanding of software development is helpful.
+
+What is more important is your ability to articulate complex ideas. Our goal is to help developers understand TinyMCE’s functionality as quickly as possible.
+
+Simply branch the [docs GitHub repository](https://github.com/tinymce/tinymce-docs) and submit a PR, or reach out to the [TinyMCE docs team](https://github.com/tinymce/tinymce-docs/issues/new?assignees=&labels=question&template=question.md) if you have a specific question.
+
+# TinyMCE documentation tech stack
+
+The documentation is built with [Antora](https://antora.org/) and text files are written in [ascii doc](https://asciidoc.org/). You will find the [repo here](https://github.com/tinymce/tinymce-docs).
+
+## Highlighting
+
+Code language syntax and associated elements are highlighted using an open and closing backtick. For example, this `object` is actually enclosed with backticks like this: `object`.
+
+Highlight words relating specifically to executable parts of software languages. Keys and their values when pertaining to objects and methods (particularly the case with TinyMCE) should have backticks. We cannot cover every use case here and contributors need to use their best judgment. What is more important is your ability to articulate complex ideas.
+
+## Code snippets
+
+Code snippets should always be complete, executable code blocks starting with the `tinymce` object. We write code snippets by enclosing the example code within three opening and closing backticks. The first set of backticks is immediately followed by the code’s language, which helps correct syntax highlighting: typically, html, css, js (and sometimes php).
```

Comment:
This section is incorrect... this isn't how code snippets are done in AsciiDoc. It uses something like this instead:
```
[source, js]
----
    tinymce.init({
      selector: 'textarea', // change this value according to your HTML
      plugins: 'codesample',
      toolbar: 'codesample',
    });
----
```

- [ ] Fix this section